scripts/deployTalentToken.ts

```diff
@@ -0,0 +1,60 @@
+import { ethers, upgrades, waffle } from "hardhat";
+import * as FactoryArtifactV2 from "../artifacts/contracts/TalentFactoryV2.sol/TalentFactoryV2.json";
+import * as TalentTokenArtifactV2 from "../artifacts/contracts/test/TalentTokenV2.sol/TalentTokenV2.json";
+import { UpgradeableBeacon__factory } from "../typechain";
+
+const { exit } = process;
+const { deployContract } = waffle;
+
+async function main() {
```

Comment:
I think this is a duplicated script: Check: [scripts/changeProdMinters.ts](https://github.com/talentprotocol/contracts/pull/54/files#diff-5793189b2da10d3cccebd73f7ed368198e8496048a219fb37d9bb53b14db54d2)